tests/test_matching.py

```diff
@@ -177,37 +189,68 @@ def test_setup_matching(self, client, test_client, helpers):
             label = 'Low'
         ))
 
-        ## Set up matching
-        conference.setup_matching()
+        # Set up reviewer matching
+        conference.setup_matching(
+            conference.client.get_group(conference.get_reviewers_id()))
+
+        reviewer_matching_ids = _matching_invitation_ids(
+            client, conference.get_reviewers_id())
+
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Program_Committee/-/Reviewing/Assignment_Configuration')
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Program_Committee/-/Reviewing/Custom_Load')
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Program_Committee/-/Reviewing/Conflict')
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Program_Committee/-/Reviewing/Aggregate_Score')
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Program_Committee/-/Reviewing/Paper_Assignment')
+
+        # Set up AC matching
+        conference.setup_matching(
+            conference.client.get_group(conference.get_area_chairs_id()))
 
-        assert client.get_invitation(id = 'auai.org/UAI/2019/Conference/-/Reviewing/Assignment_Configuration')
-        assert client.get_invitation(id = 'auai.org/UAI/2019/Conference/-/Reviewing/Custom_Load')
-        assert client.get_invitation(id = 'auai.org/UAI/2019/Conference/-/Reviewing/Conflict')
-        assert client.get_invitation(id = 'auai.org/UAI/2019/Conference/-/Reviewing/Aggregate_Score')
-        assert client.get_invitation(id = 'auai.org/UAI/2019/Conference/-/Reviewing/Paper_Assignment')
+        ac_matching_ids = _matching_invitation_ids(
+            client, conference.get_area_chairs_id())
+
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Senior_Program_Committee/-/Reviewing/Assignment_Configuration')
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Senior_Program_Committee/-/Reviewing/Custom_Load')
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Senior_Program_Committee/-/Reviewing/Conflict')
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Senior_Program_Committee/-/Reviewing/Aggregate_Score')
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Senior_Program_Committee/-/Reviewing/Paper_Assignment')
 
         bids = client.get_edges(invitation = conference.get_bid_id())
         assert bids
         assert 6 == len(bids)
 
-        custom_loads = client.get_edges(invitation = 'auai.org/UAI/2019/Conference/-/Reviewing/Custom_Load')
+        custom_loads = []
+        custom_loads.extend(
```

Comment:
why do you need to append both results into an array? why don't you assert each result independently?

Reply:
I don't need to, this was just my attempt to keep things as similar as possible to the way it was before. Should I change it?

Reply:
I changed it